gateway-js/src/__tests__/gateway/reporting.test.ts

```diff
@@ -1,12 +1,9 @@
 import { gunzipSync } from 'zlib';
 import nock from 'nock';
 import { GraphQLSchemaModule } from 'apollo-graphql';
-import gql from 'graphql-tag';
 import { buildSubgraphSchema } from '@apollo/subgraph';
 import { ApolloServer } from 'apollo-server';
 import { ApolloServerPluginUsageReporting } from 'apollo-server-core';
```

Comment:
I had a similar desire to remove usage of `apollo-link` in main, but I ended up including the functions from the supported library `@apollo/client` as a devDependency. Would it make sense to port this to main as well so we can eliminate it?

Reply:
Definitely plan to port this! (I'll be sticking to @abernix's port/backport label convention going forward, I think that's helpful for signaling intent)